Sources/Commands/SwiftTestTool.swift

```diff
@@ -88,42 +95,44 @@ public struct SwiftTestTool {
             if let dir = opts.chdir {
                 try chdir(dir)
             }
-        
+
+            let configuration = "debug"  //FIXME should swift-test support configuration option?
+            func determineTestPath() throws -> String {
+                //FIXME better, ideally without parsing manifest since
+                // that makes us depend on the whole Manifest system
```

Comment:
You could invoke `swift build` with `--dump-package` in JSON and read the name from there :)